src/main/java/uk/gov/hmcts/darts/task/runner/impl/ProcessDetsToArmResponseAutomatedTask.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.darts.task.runner.impl;
```

Comment:
We typically add some tests for a new automated task, you can see an example if this (old, outdated now :( reference PR we have: https://github.com/hmcts/darts-api/pull/1559/files)

Reply:
As we dynamically load these tasks now instead of creating the wrapper code manually, I don't think there are any tests required :)

Reply:
pardon me, I was not aware that my reference PR is now obsolete after the simplifications we've done to the framework. You can ignore my comments then